develop/dev-guide-third-party-support.md

```diff
@@ -32,33 +32,153 @@ PingCAP 与开源社区合作，通过三方工具提供以下支持：
 | JavaScript | [mysql](https://github.com/mysqljs/mysql) | v2.18.1 | Compatible | N/A | N/A |
 | PHP | [mysqlnd](https://dev.mysql.com/downloads/connector/php-mysqlnd/) | PHP 5.4+ | Compatible | N/A | N/A |
 | Python | [MySQL Connector/Python](https://downloads.mysql.com/archives/c-python/) | 8.0 | Compatible | N/A | N/A |
 
 ## ORM
 
-| 编程语言 | ORM 框架 | 最新已测试版本 | 支持等级 | TiDB 适配器 | 教程 |
-| - | - | - | - | - | - |
-| Go | [gorm](https://github.com/go-gorm/gorm) | v1.23.5 | Full | N/A | [TiDB 和 Golang 的简单 CRUD 应用程序](/develop/dev-guide-sample-application-golang.md) |
-| Go | [beego](https://github.com/beego/beego) | v2.0.3 | Full | N/A | N/A |
-| Go | [upper/db](https://github.com/upper/db) | v4.5.2 | Full | N/A | N/A |
-| Go | [xorm](https://gitea.com/xorm/xorm) | v1.3.1 | Full | N/A | N/A |
-| Java | [Hibernate](https://hibernate.org/orm/) | 6.1.0.Final | Full | N/A | [TiDB 和 Java 的简单 CRUD 应用程序](/develop/dev-guide-sample-application-java.md) |
-| Java | [MyBatis](https://mybatis.org/mybatis-3/) | v3.5.10 | Full | N/A | [TiDB 和 Java 的简单 CRUD 应用程序](/develop/dev-guide-sample-application-java.md) |
-| Java | [Spring Data JPA](https://spring.io/projects/spring-data-jpa/) | 2.7.2 | Full | N/A | [使用 Spring Boot 构建 TiDB 应用程序](/develop/dev-guide-sample-application-spring-boot.md) |
-| Java | [jOOQ](https://github.com/jOOQ/jOOQ) | v3.16.7 (Open Source) | Full | N/A | N/A |
-| Ruby | [Active Record](https://guides.rubyonrails.org/active_record_basics.html) | v7.0 | Full | N/A | N/A |
-| JavaScript/TypeScript | [sequelize](https://www.npmjs.com/package/sequelize) | v6.20.1 | Compatible | N/A | N/A |
-| JavaScript/TypeScript | [Knex.js](https://knexjs.org/) | v1.0.7 | Compatible | N/A | N/A |
-| JavaScript/TypeScript | [Prisma Client](https://www.prisma.io/) | 3.15.1 | Compatible | N/A | N/A |
-| JavaScript/TypeScript | [TypeORM](https://www.npmjs.com/package/typeorm) | v0.3.6 | Compatible | N/A | N/A |
-| PHP | [laravel](https://laravel.com/) | v9.1.10 | Compatible | [laravel-tidb](https://github.com/colopl/laravel-tidb) | N/A |
-| Python | [Django](https://pypi.org/project/Django/) | v4.0.5 | Compatible | [django-tidb](https://github.com/pingcap/django-tidb) | N/A |
-| Python | [peewee](https://github.com/coleifer/peewee/) | v3.14.10 | Compatible | N/A | N/A |
-| Python | [PonyORM](https://ponyorm.org/) | v0.7.16 | Compatible | N/A | N/A |
-| Python | [SQLAlchemy](https://www.sqlalchemy.org/) | v1.4.37 | Compatible | N/A | N/A |
+<table>
+   <thead>
+      <tr>
+         <th>编程语言</th>
+         <th>ORM 框架</th>
+         <th>最新已测试版本</th>
+         <th>支持等级</th>
+         <th>TiDB 适配器</th>
+         <th>教程</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td rowspan="4">Go</td>
+         <td><a href="https://github.com/go-gorm/gorm" target="_blank" referrerpolicy="no-referrer-when-downgrade">gorm</a></td>
+         <td>v1.23.5</td>
+         <td rowspan="4">Full</td>
```

Comment:
I think we don't need to enable rowspan for the `support level` column. Just keep the structure of the table simple.

Reply:
Got it.